_data/en/settings.yml

```diff
@@ -1,12 +1,3 @@
-404_page:
```

Comment:
While it'd be nice to translate the 404 page, in practice Federalist doesn't support it, since it's very simply looking for a `404.html` page in the root of the site directory ([reference](https://federalist.18f.gov/documentation/customization/#custom-404-pages)). Therefore, even if we generate locale-specific 404 pages, they won't be used.

Example: https://login.gov/es/abcd